docs/oscar_references.bib

```diff
@@ -318,3 +318,42 @@ @inproceedings{BDLP19
   HAL_ID = {hal-02912148},
   HAL_VERSION = {v1},
 }
+
+@Book{DK98,
+ Author = {Harm Derksen and Gregor Kemper},
+ Title = {Computational invariant theory. With two appendices by Vladimir L. Popov and an addendum by Nobert A. Campo and Vladimir L. Popov. 2nd enlarged edition},
+ Edition = {2nd enlarged edition},
+ ISBN = {978-3-662-48420-3/hbk; 978-3-662-48422-7/ebook},
+ Pages = {xxii + 366},
+ Year = {2015},
+ Publisher = {Berlin: Springer},
+ Language = {English},
+ MSC2010 = {13-00 13-02 13A50 14L24 13P10 13-04},
+ Zbl = {1332.13001}
+}
+
+@InCollection{DJ98,
+ Author = {Wolfram Decker and Theo de Jong},
+ Title = {Gr\"obner bases and invariant theory},
+ BookTitle = {Gr\"obner bases and applications. Based on a course for young researchers, January 1998, and the conference ``33 years of Gr\"obner bases'', Linz, Austria, February 2--4, 1998},
+ ISBN = {0-521-63298-6/pbk},
+ Pages = {61--89},
+ Year = {1998},
+ Publisher = {Cambridge: Cambridge University Press},
+ Language = {English},
+ MSC2010 = {13P10 15A72 13A50 14L24},
+ Zbl = {0908.13019}
+}
+
+@InCollection{DGP99,
+ Author = {Wolfram Decker and Gert-Martin Greuel and Gerhard Pfister},
+ Title = {Primary decomposition: Algorithms and comparisons},
+ BookTitle = {Algorithmic algebra and number theory. Selected papers from a conference, Heidelberg, Germany, October 1997},
+ ISBN = {3-540-64670-1/hbk},
+ Pages = {187--220},
+ Year = {1999},
+ Publisher = {Berlin: Springer},
+ Language = {English},
+ MSC2010 = {13P10 13A15 13F20 68W30 13-04},
+ Zbl = {0932.13019}
+}
```

Comment:
```suggestion
}

```

Reply:
what?

Reply:
All files should ideally end with a newline. Not important